test-case-creation-strategy.mdx

```diff
@@ -0,0 +1,189 @@
+---
+title: Test case creation
+description: "How to best create test cases for your project"
+icon: "lightbulb"
+---
+
+## General Principles
+
+- start with a prompt
```

Comment:
it's not really starting with a prompt from the sign-up and proposal though, should this just be `start with our AI` or something? also underneath